todocli/todo_api/queries.py

```diff
@@ -0,0 +1,76 @@
+from typing import Union
+
+
+from todocli.rest_request import RestRequestGet
+from todocli.todo_api import api_urls
+from todocli.todo_api.exceptions import (
+    TaskNotFoundByName,
+    TaskNotFoundByIndex,
+    ListNotFound,
+)
+
+list_ids_cached = {}
+
+
+def query_tasks(list_name: str, include_completed=False, num_tasks: int = 100):
+    query_url = api_urls.get_tasks(
+        get_list_id_by_name(list_name), num_tasks, include_completed=include_completed
+    )
+
+    return RestRequestGet(query_url).execute()
+
+
+def query_task(list_name: str, task_name: str):
+    query_url = api_urls.get_task_by_name(get_list_id_by_name(list_name), task_name)
+    try:
+        return RestRequestGet(query_url).execute()[0]
```

Comment:
What is contained in the list/tuple returned from `RestRequestGet(query_url).execute()`? Is it possible to move the index lookup inside `execute`?